This is a temporal fix for version 1.3.x when compiling with Go 1.13+. The change
was already applied upstream and will be part of newer Vault versions:
https://github.com/hashicorp/vault/commit/fa245e6a1db4bd8b8328c6c1a9850269078a5da9
diff --git a/go.mod b/go.mod
index d4bb30da1..a63584817 100644
--- a/go.mod
+++ b/go.mod
@@ -2,8 +2,6 @@ module github.com/hashicorp/vault

 go 1.12

-replace git.apache.org/thrift.git => github.com/apache/thrift v0.12.0
-
 replace github.com/hashicorp/vault/api => ./api

 replace github.com/hashicorp/vault/sdk => ./sdk
